gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/PickerPopup.kt

```diff
@@ -100,9 +103,15 @@ private fun PickerPopup(
                         shadowElevation = 2.dp,
                     ) {
                         LazyColumn {
-                            itemsIndexed(popupButtons) { index, item ->
-                                item()
-                                if (index < popupButtons.size - 1) {
+                            itemsIndexed(popupItems) { index, item ->
+                                PopupButton(
+                                    text = stringResource(item.text),
+                                    iconRes = item.iconRes,
+                                    contentDescription = stringResource(item.contentDescription),
+                                    shape = RoundedCornerShape(topStart = 8.dp, topEnd = 8.dp),
```

Comment:
This should be different for the first, middle, and last items. You can see it's incorrect when you open the `MediaPickerPopup` and tap&hold the `Take Photo` button. The ripple will have rounded corners at the top not matching the shape of the button.

Reply:
Good catch! Updated!